node/src/reactor/participating/tests.rs

```diff
@@ -369,6 +369,13 @@ async fn run_equivocator_network() {
         .map(|era_number| switch_blocks.bids(net.nodes(), era_number))
         .collect();
 
+    // Since this setup sometimes fails to produce an equivocation we return early here.
+    // TODO: Remove this once https://github.com/casper-network/casper-node/issues/1859 is fixed.
+    if switch_blocks.equivocators(0).is_empty() {
+        error!("Failed to equivocate in the first era.");
```

Comment:
People will have to remember to check the logs for occurences of `ERROR`-level entries before merging?

Reply:
If they don't that's not a big issue, though: Most of the time we do manage to create an equivocation here, even in CI, and then the test works correctly, and would actually _fail_ if something unexpected happened.

Anyway, I don't have a better idea currently, and this is only a temporary workaround. I think it's better than ignoring the test entirely.

Reply:
I agree about not having a better idea. I was just wondering how we would catch a case when we break something in the code and never produce/detect an equivocation. Then, the only way to find out about it would be to run the test repeatadly and check if it logs error every time?

Reply:
Yes, that's the danger; that's why this PR doesn't close https://github.com/casper-network/casper-node/issues/1859.
In the meantime, as a temporary workaround, this is still better than having the complete test ignored.